src/components/UppyCdnExample.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import CodeBlock from '@theme/CodeBlock';
+import Admonition from '@theme/Admonition';
+
+
+const uppyVersion = '2.9.0' // todo
+
+export default function UppyCdnExample ({
+  children,
+  uppyCssName = 'uppy.min.css',
+  uppyJsName = 'uppy.min.js',
+}) {
+  let lines = [];
+
+  React.Children.toArray(children).forEach((child) => {
+    lines = [...lines, ...String(child).trim().split('\n').map((line) => line.trim())];
+  })
```

Comment:
I'm surprised this is necessary, if we pass a correct string with newlines and everything, won't it look good?

Reply:
Yes, but I think it's nice if the developer doesn't have to worry about providing the correct indentation everytime we want to use the component.